src/webpack/webpackTools.ts

```diff
@@ -76,7 +79,7 @@ export const manifestPlugin = (() => {
         from: './res/provider/app.json',
         to: '.',
         transform: (content) => {
-            const config = JSON.parse(content);
+            const config = JSON.parse(content.toString());
```

Comment:
I might be missing something, but why do you need this `toString` here?

Reply:
Webpack types changed it to a buffer.  I feel like this change should already be in the project but it appears not.

Reply:
Hmm. Fair enough.